src/Control/Middleware/ExecMetricMiddleware.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace SilverStripe\Control\Middleware;
+
+use SilverStripe\Assets\File;
+use SilverStripe\Control\Director;
+use SilverStripe\Control\HTTPRequest;
+use SilverStripe\Control\HTTPResponse;
+use SilverStripe\Dev\Debug;
+
+/**
+ * Secures requests by only allowing a whitelist of Host values
```

Comment:
Is that really what this middleware does?

Reply:
Oh whoops that's a retrospective review. Explains why I didn't see all the options when submitting it.

Reply:
:man_facepalming: I copied over one of the other files. Forgot to update the comment.

Reply:
I'll do a follow up PR.

Reply:
https://github.com/silverstripe/silverstripe-framework/pull/8770